packages/@react-spectrum/breadcrumbs/src/Breadcrumbs.tsx

```diff
@@ -132,38 +132,39 @@ function Breadcrumbs<T>(props: SpectrumBreadcrumbsProps<T>, ref: DOMRef) {
 
   useLayoutEffect(updateOverflow, [children, updateOverflow]);
 
-  useLayoutEffect(() => {
-    // Only try to restore focus after an action.
-    if (!isAfterAction.current) {
-      return;
-    }
-    isAfterAction.current = false;
+  // useLayoutEffect(() => {
+  //   // Only try to restore focus after an action.
+  //   if (!isAfterAction.current) {
+  //     return;
+  //   }
+  //   isAfterAction.current = false;
 
-    // Wait a frame for updateOverflow.
-    requestAnimationFrame(() => {
-      if (
-        // Where breadcrumbs exist, 
-        domRef.current &&
-        // the current breadcrumb is defined,
-        currentRef.current &&
-        // and the current focus has either been removed from the DOM,
-        // or is within the breadcrumbs,
-        (
-          document.activeElement === document.body ||
-          domRef.current.contains(document.activeElement)
-        )
-      ) {
-        // focus the current breadcrumb.
-        currentRef.current.UNSAFE_getDOMNode().focus();
-      }
-    });
-  }, [domRef, currentRef, isAfterAction, children]);
+  //   // Wait a frame for updateOverflow.
+  //   requestAnimationFrame(() => {
+  //     if (
+  //       // Where breadcrumbs exist,
+  //       domRef.current &&
+  //       // the current breadcrumb is defined,
+  //       currentRef.current &&
+  //       // and the current focus has either been removed from the DOM,
+  //       // or is within the breadcrumbs,
+  //       (
+  //         document.activeElement === document.body ||
+  //         domRef.current.contains(document.activeElement)
+  //       )
+  //     ) {
+  //       // focus the current breadcrumb.
+  //       currentRef.current.UNSAFE_getDOMNode().focus();
+  //     }
+  //   });
+  // }, [domRef, currentRef, isAfterAction, children]);
 
   function triggerOnAction(key: Key) {
     if (onAction) {
       onAction(key);
-      isAfterAction.current = true;
+      // isAfterAction.current = true;
```

Comment:
1. This would only work if there is actually an `onAction` prop passed by the user.
2. It might be problematic in controlled scenarios. We are basically assuming that the breadcrumbs will rerender immediately here, but that might not always happen. Then if they updated some time in the future we'd trigger a focus because this flag would still be true.